src/main/java/com/redhat/coolstore/service/ShoppingCartService.java

```diff
@@ -14,7 +14,7 @@
 import com.redhat.coolstore.model.ShoppingCartItem;
 
 @Stateful
```

Comment:
In this case it has done the namespace changes but not this annotation change. The iterative approach that we talked about should handle this @jwmatthews